main.go

```diff
@@ -290,6 +290,10 @@ func initContext(ctx context.Context, cfg *config.Config) context.Context {
 	// disable this for all terms on this platform
 	if runtime.GOOS == "windows" {
 		color.NoColor = true
+	} else {
+		if cfg.Get("core.nocolor") != "" {
+			color.NoColor = cfg.GetBool("core.nocolor")
+		}
```

Comment:
```suggestion
		color.NoColor = cfg.GetBool("core.nocolor")
```
If I understand it correctly, the

Reply:
Right now I'm not sure. I had a reason to add this.

But definitely two TODOs for me here:
* document why we're checking if it's set, if we decide to keep it
* add an IsSet method to the config to make clear what we're doing (better readable than comparing to an empty string, which is likely not obvious)

Reply:
There is no default, so I still want to check if it's set.